tests/cypress/cypress/integration/Package_Dashboard_Tabs.spec.feature

```diff
@@ -108,5 +108,5 @@ Feature: OY2-14464 Package Dashboard - Separate Tab for Waivers and SPAs
         When Login with state submitter user
         And click on Packages
         And click on the Waivers tab
-        And search for "MD.1117"
+        And search for "MD-33463.R00.00"
```

Comment:
oh, glad to see... was thinking about getting rid of the . waiver ids in the seed data :)